gammapy/datasets/io.py

```diff
@@ -472,3 +476,172 @@ def read(self):
             )
 
         return SpectrumDatasetOnOff(name=name, exposure=exposure, **kwargs)
+
+
+class FermipyDatasetsReader(DatasetReader):
+    """Create datasets from Fermi-LAT configuration file.
+
+    Parameters
+    ----------
+    filename : str
+        Configuration file path
+    edisp_bins : int
+        Number of margin bins to slice in energy. Default is 0.
+        For now only maps created with edisp_bins=0 in fermipy configuration are supported,
+        in that case the emin/emax in the fermipy configuration will correspond to the true energy range for gammapy,
+        and  a value edisp_bins>0 should be set here in order to apply the energy dispersion correctly.
+        With a binning of 8 to 10 bins per decade, it is recommended to use edisp_bins ≥ 2
+        (See https://fermi.gsfc.nasa.gov/ssc/data/analysis/documentation/Pass8_edisp_usage.html)
+
+    """
+
+    tag = "fermipy"
+
+    def __init__(self, filename, edisp_bins=0):
+        self.filename = make_path(filename)
```

Comment:
Somehow, for reading a file, I think its easier to specify the filename in the function rather than through the class

Reply:
I used the same logic than for the OGIPDatasetReader, so if we change that in should be modified consitently for all Reader class  in a separated PR.
On a longer term I expect the mismatch between true energy axis of the different irfs for edisp_bins>0 to be fixed on the fermipy side. Then won't need that option and so user won't have to use this `FermiDatasetsReader` but would use directly  `Datasets.read(format="fermipy")`.